src/raw.rs

```diff
@@ -0,0 +1,87 @@
+#![allow(non_camel_case_types)]
+
+use pgx::*;
+
+// TODO: Is this the right approach to declaring `bytea` and `TimestampTz`?
+extension_sql!(
+    r#"
+        SELECT 1;
+    "#,
```

Comment:
You don't need actual contents you can just do
```suggestion
    ""
```
but otherwise 👍

Reply:
👍